src/main/java/com/google/sps/servlets/YoutubePost.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+public class YoutubePost {
+  String title;
+  String content;
+  String url;
+  String id;
+
+  @Override
+  public String toString() {
+    String post = ("title: " + title + "\n"
```

Comment:
Why does this method only return a String with all the information?

Reply:
Good question! It's [Java's toString method](https://www.javatpoint.com/understanding-toString()-method#:~:text=The%20toString()%20method%20returns,depends%20on%20your%20implementation.).

Reply:
Oh, I think I phrased my question wrong. Why is this class only returning this String and not using any methods returning the information?

Reply:
It shows all of the information for debugging purposes since its where the data model is populated for later use. It fits nicely into the response println in Data Servlet to pass newPost as a parameter.